revolv/static/project.scss

```diff
@@ -77,3 +476,227 @@
     height: 300px;
     margin: 0 20px 20px 20px;
 }
+
+// --- Mobile Responsiveness Styles ------------------------------------------------------
+// TO FIX @MEDIA HEADERS AND TO ADD IN PORTRAIT/ORIENTATION STUFF
+
+@media #{$small-only} {
+    .project-updates {
+
+        .columns {
+            padding-left: 0px;
+            padding-right: 0px;
+        }
+
+        .update-container {
+            margin: 10px 0px;
+
+            .date-container {
+                text-align: left;
+                margin-bottom: 50px;
+
+            }
+
+            .update-text {
+                font-weight: 500;
+                font-size: 1rem;
+            }
+        }
+    }
+    .map {
+
+        height: 250px;
```

Comment:
This might make a good variable e.g. `$map-height`, you're using it in a few different places